functions/backup.bash

```diff
@@ -413,6 +413,11 @@ mirror_SD() {
 ##   setup_mirror_SD()
 ##
 setup_mirror_SD() {
+  if [[ -n "$UNATTENDED" ]] && [[ -z "$backupdrive" ]]; then
```

Comment:
please separate features and features from fixes next time. This shouldn't be in this PR.

Reply:
K